specs/_template/04_other.rst

```diff
@@ -0,0 +1,114 @@
+=======  ==================  ======  ==========
+ epic          title         author   created
+=======  ==================  ======  ==========
+`EPIC`_  SPECIFICATION_NAME  AUTHOR  CREATED_AT
+=======  ==================  ======  ==========
+
+.. _EPIC: gnosis/safe#EPIC
```

Comment:
For simplicity I would remove this completely.

Reply:
It's a lot of boiler plate. Could we move some of this into a general place so we don't have to copy it over and over again?

Reply:
If some of the infos are required I would put them in a technical section (proposed in another comment)

Reply:
👍

Reply:
@tschubotz we can remove the boilerplate code, but GitHub prohibits "include" directives because of security issues on their side. IF we would generate a doc (like readthedocs) from these specifications, then we can reduce duplication and do all the nice stuffs.

Reply:
I would prefer not to use a readthedocs. This is overly complex in my opinion. If we really write specs that are so long that we need a readthedocs we should maybe also think about if we should split them up into smaller specs.